core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -30,8 +30,22 @@ public interface Configs {
 
   String ENTITY_KV_ROCKSDB_BACKEND_PATH_KEY = "gravitino.entity.store.kv.rocksdbPath";
 
-  Long DEFAULT_KV_DELETE_AFTER_TIME = 604800000L; // 7 days
-  String KV_DELETE_AFTER_TIME_KEY = "gravitino.entity.store.kv.deleteAfterTimeMs";
+  // Config for data keep time after soft deletion, in milliseconds.
+  String STORE_DELETE_AFTER_TIME_KEY = "gravitino.entity.store.deleteAfterTimeMs";
```

Comment:
If you change the key of the config option directly, this is a incompatible change.
It's ok before we release 1.0. Because we only guarantee that compatibility after 1.0.
More common way is that we change the old config option to deprecated and add new config option.

Reply:
thx, i have updated the changes by deprecating the old config.

Reply:
You should use the method `deprecated` in the config option instead of `annotattion`

Reply:
You can find it in the ConfigBuidler#90L

Reply:
> You can find it in the ConfigBuidler#90L

got it, done, thx for your patience.